src/nodes/devices-eurotronic.js

```diff
@@ -29,7 +29,7 @@ module.exports = function (RED) {
         }
 
         utils.setConnectionState(bridgeNode, node);
-        bavaria.observer.register(bridgeNode.id + "_connected", function (message) {
+        let regId = bavaria.observer.register(bridgeNode.id + "_connected", function (message) {
```

Comment:
I believe these can be made const in all files.... If you want.

Reply:
That's what I've done. If you see one I don't changed, let me know.

Reply:
Sorry I skipped the word const while reading ^^